src/core/rewrite.ml

```diff
@@ -58,18 +59,6 @@ let eq : ctxt -> term -> term -> bool = fun ctx a b -> a == b ||
   in
   try eq [(a,b)]; true with Not_equal -> false
 
-(** Rewrite patterns as in Coq/SSReflect. See "A Small Scale
-    Reflection Extension for the Coq system", by Georges Gonthier,
-    Assia Mahboubi and Enrico Tassi, INRIA Research Report 6455, 2016,
-    http://hal.inria.fr/inria-00258384, section 8, p. 48. *)
-type rw_patt =
-  | RW_Term           of term
-  | RW_InTerm         of term
-  | RW_InIdInTerm     of (term, term) Bindlib.binder
-  | RW_IdInTerm       of (term, term) Bindlib.binder
-  | RW_TermInIdInTerm of term * (term, term) Bindlib.binder
-  | RW_TermAsIdInTerm of term * (term, term) Bindlib.binder
-
```

Comment:
As I already said, I think this should stay here. This is only relevant for `rewrite` and related future tactics.